index.css

```
@@ -0,0 +1,114 @@
+input.swap:checked ~ main
```

В селекторе класса не обязательно указывать тег
```suggestion
.swap:checked ~ main
```